index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<meta http-equiv="X-UA-Compatible" content="ie=edge" />
+		<title>Prompt Calculator</title>
+		<style>
+			body {
+				height: 100vh;
+				display: flex;
+				align-items: center;
+				background: linear-gradient(to top left, #28b487, #7dd56f);
+			}
+			h1 {
+				font-family: sans-serif;
+				font-size: 50px;
+				line-height: 1.3;
+				width: 100%;
+				padding: 30px;
+				text-align: center;
+				color: white;
+			}
+		</style>
+	</head>
+	<body>
+		<h1>Prompt Calculator</h1>
+		<script src="script.js"></script>
+	</body>
```

Comment:
@ManSOorcode Can you explain why are you not putting script tags at the top in head?

Reply:
when we place JavaScript at the bottom of html, which give time to load html before any JavaScript loads and prevent error and also improve website response time.